app/services/decisions/outgoings_decision_tree.rb

```diff
@@ -45,5 +55,12 @@ def after_answers
     def crime_application
       form_object.crime_application
     end
+
+    def partner_relevant
+      return if partner_detail.nil?
+
+      partner_detail.involvement_in_case == 'none' ||
+        (partner_detail.involvement_in_case == 'codefendant' && partner_detail.conflict_of_interest == 'no')
+    end
   end
```

Comment:
The `include_partner_in_means_assessment?` method in the Type of means assessment module would take care of this check for you